website/content/docs/custom-widgets.md

```diff
@@ -198,3 +202,92 @@ You can also return a promise from `isValid`. While the promise is pending, the
 ```
 
 **Note:** Do not create a promise inside `isValid` - `isValid` is called right before trying to persist. This means that even if a previous promise was already resolved, when the user hits 'save', `isValid` will be called again. If it returns a new promise, it will be immediately marked as "in error" until the new promise resolves.
+
+## Creating a custom widget using JSX syntax with a build step
+
+Widgets are inputs for the Netlify CMS editor interface. It's a React component that receives user input and outputs a serialized value. Those are the only rules - the component can be extremely simple, like text input, or extremely complicated, like a full-blown markdown editor. They can make calls to external services, and generally do anything that JavaScript can do.
+
+We have created a boilerplate for creating Netlify CMS widgets. You can check the [netlify-cms-widget-starter](https://github.com/netlify/netlify-cms-widget-starter) out for a starter project to fork. The starter currently provides a basic string widget, which you can customize, extend, or replace entirely with your own widget.
+
+### [](https://github.com/netlify/netlify-cms-widget-starter#getting-started)Getting started
```

Comment:
We would like to drop https://github.com/netlify/netlify-cms-widget-starter.git completely.
Meaning we should detail in the docs the actions required to set up that repo.
A very slim version would be:

1. Create a directory `mkdir my-custom-widget`.
2. Navigate to the directory `cd my-custom-widget`.
3. Run `npm init`. 
4. Create a `src` directory with the files `Control.js`, `Preview.js` and `index.js`.
5. Install the following dependencies...
6. Create a Webpack configuration file...

and so on